__tests__/app.test.js

```diff
@@ -282,3 +280,31 @@ test(" response with 204 and deletes the comment", () => {
       });
     });
 });
+
+describe("users endpoint ", () => {
+  test.only("response with the status of 200 and array of all the available users ", () => {
```

Comment:
Please do not submit pr's with .only's/.skip's to ensure all tests are passing correctly